owasp-dependency-check-suppressions.xml

```diff
@@ -157,6 +157,13 @@
     <packageUrl regex="true">^pkg:maven/com\.nimbusds/nimbus\-jose\-jwt@4.41.1$</packageUrl>
     <cve>CVE-2019-17195</cve>
   </suppress>
+  <suppress>
+    <notes><![CDATA[
```

Comment:
Can you add a comment that this is false positive?

Reply:
Done